### PR TITLE
Update Nuget from net45 to net46

### PR DESCRIPTION
**Pull request by PatoBeltran**
Moving NuGet projects to use net46 instead of net45

### CODE REVIEW

File: src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj]: Revert this as there are no other changes.

File: src/NuGet.Core/NuGet.Build.Tasks/NuGet.Build.Tasks.csproj [hunk 2]
```diff
@@ -56,5 +56,5 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <Import Project="$(BuildCommonDirectory)common.targets"/>
+  <Import Project="$(BuildCommonDirectory)common.targets" />
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.Build.Tasks/NuGet.Build.Tasks.csproj]: nit: extra space.

File: src/NuGet.Core/NuGet.Versioning/NuGet.Versioning.csproj [hunk 3]
```diff
@@ -34,5 +34,5 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <Import Project="$(BuildCommonDirectory)common.targets"/>
+  <Import Project="$(BuildCommonDirectory)common.targets" />
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.Versioning/NuGet.Versioning.csproj]: nit: extra space.

**Review by nkolev92 — APPROVED** [review]
minor feedback. 

👏

**Review by emgarten — APPROVED** [review]
LGTM, I didn't realize the rest of the projects were already net46 such as the VSIX.

**Pull request merged**